package.json

```
@@ -34,7 +34,7 @@
     },
     "dependencies": {
         "pxt-common-packages": "8.1.11",
-        "pxt-core": "6.2.26"
+        "pxt-core": "6.2.27"
```

```suggestion
        "pxt-core": "6.2.28"
```

If we could bump this to pick up https://github.com/microsoft/pxt/pull/7707 that would be nice~